Jenkinsfile.release

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2017-present Sonatype, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@Library(['private-pipeline-library', 'jenkins-shared']) _
+
+import com.sonatype.jenkins.shared.Expectation
+
+properties([
+  parameters([
+    run(name: 'releaseBuild',
+        filter: 'SUCCESSFUL',
+        projectName: 'insight/insight-brain/release',
+        description: 'The latest release of IQ Server to build the docker image')
+  ])
+])
+
+String imageName = 'sonatype/nexus-iq-server',
+String version = ''
+String checksum = ''
+
+dockerizedBuildPipeline(
+  prepare: {
+    githubStatusUpdate('pending')
+    version = getVersionFromBuildName(env.releaseBuild_NAME)
+    checksum = readBuildArtifact('insight/insight-brain/release', env.releaseBuild_NUMBER, "artifacts/nexus-iq-server-${version}-bundle.tar.gz.sha256").trim()
+    updateIQServerVersionAndChecksum(version, checksum)
+    commitAndPushChanges(version)
+  },
+  setVersion: {
+    env['VERSION'] = version.split('-')[0]
```

Comment:
For some reason the current build removes the build number from the version string e.g. "1.100.0-01" becomes "1. 100.0-01". I think we should keep it for consistency (since iq has already been published in docker hub)

Reply:
This is used as a Docker tag. Now this images has tags like `1.158.0` in DockerHub. The slim image tags look like `1.158.0-slim`. I guess we want to keep this format going forward.

Reply:
> This is used as a Docker tag. Now this images has tags like `1.158.0` in DockerHub. The slim image tags look like `1.158.0-slim`. I guess we want to keep this format going forward.

Sorry I realised my above [comment](https://github.com/sonatype/docker-nexus-iq-server/pull/120#discussion_r1155802392) was incorrect. I've updated it. Yes you're correct the docker tag will use the same format it has been using e.g. 1.100.0.